hudi-examples/pom.xml

```diff
@@ -194,6 +194,7 @@
     <dependency>
       <groupId>org.apache.avro</groupId>
       <artifactId>avro</artifactId>
+      <version>${avro.version}</version>
```

Comment:
same question: do we need this? thought it inherits the same version from the root pom